i18n/translations/en.json

```diff
@@ -769,7 +769,8 @@
 	"user.changeLanguage": "Change language",
 	"user.changeName": "Rename this user",
 	"user.changePassword": "Change password",
-	"user.changePassword.current": "Your own password",
+	"user.changePassword.admin": "Your own password",
```

Comment:
Suggestion: Couldn't we just call it `user.changePassword.own` to not bind it to a role? There could be roles later which are also allowed to change passwords.